puppet/modules/consular/manifests/init.pp

```diff
@@ -0,0 +1,39 @@
+class consular(
+  $consular_args = [],
+) {
+  class { 'python' :
+    version    => 'system',
+    pip        => 'present',
+    dev        => 'present',
+    virtualenv => 'present',
+    gunicorn   => 'absent',
+  }
+
+  package { [
+    'build-essential',
+    'libssl-dev',
+    'libffi-dev',
+    'openssl',
+    'libffi6',
+  ]: }
+  ->
```

Comment:
`->` is a one-time thing, `~>` is triggers a refresh every time the thing on the left changes.